strategies/categorized/src/main/java/org/arquillian/smart/testing/strategies/categorized/TagsParser.java

```diff
@@ -0,0 +1,107 @@
+package org.arquillian.smart.testing.strategies.categorized;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.arquillian.smart.testing.logger.Log;
+import org.arquillian.smart.testing.logger.Logger;
+
+class TagsParser extends AbstractParser {
+
+    public static final String TAG = "org.junit.jupiter.api.Tag";
+    public static final String TAGS = "org.junit.jupiter.api.Tags";
```

Comment:
Can't we make `TAG` and `TAGS` private?

Reply:
yeah they are public because they were needed in previous design and I miss to change. Ty